client/state/selectors/get-site-options.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { getRawSite } from 'state/sites/selectors';
+
+/**
+ * Returns the site options
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}    siteId   Site ID
+ * @returns  {?Object}   options  Site options or null
+ */
+export default ( state, siteId ) => {
+	const site = getRawSite( state, siteId );
+	if ( ! site ) {
+		return null;
+	}
+	const options = site.options ? { ...site.options } : {};
```

Comment:
Hi @aduth, site.options is false if options are not set e.g: undefined. I think your suggestion is great and increases the performance of our selector so I applied it.  Thank you!